applications_tests/lethe-fluid/heat_transfer_average_temperature_modified_initial_time.mpirun=1.output

```diff
@@ -0,0 +1,48 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       1
+   Number of degrees of freedom: 12
+   Volume of triangulation:      0.001
+   Number of thermal degrees of freedom: 4
+************************
+---> Simulation Restart 
```

Comment:
This is a test with a restart file. Could you please add the generator of the restart file inside of the genrator folder so that if something changes the restart files can be remade.